src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font for `Roboto` to ensure proper font rendering across different browsers. For example, use `font-family: 'Roboto', sans-serif;`.